code/style.css

```diff
@@ -1,7 +1,398 @@
+}
+
+.custom-form-container input:checked ~ .custom-input{
+  background: #444;
+}
+
+.custom-form-container input:checked ~ .custom-input:after {
+  display: block;
+}
+
+.custom-radio:after {
+  content: "";
+  position: absolute;
+  display: none;
+}
+
+.custom-radio {
+  border-radius: 50%;
+}
+
+.custom-form-container .custom-radio:after {
+ top: 6px;
+ left: 6px;
+ width: 7px;
+ height: 7px;
+ border-radius: 50%;
+ background: white;
+}
+
+.wrapper-checkbox-row {
+  display: flex;
+  align-self: flex-start;
+  margin-bottom: 20px;
+}
+
+.custom-checkbox {
+  border-radius: 4px;
+}
+
+.custom-checkbox::after {
+  content: "";
+  position: absolute;
+  width: 3px;
+  height: 6px;
+  border: solid #fff;
+  border-width: 0 3px 3px 0;
+  top: 4px;
+  left: 6px;
+  transform: rotate(45deg);
+  display: none;
+}
+
+.form-input[type="submit"] {
+  align-self: flex-end;
+  border: none;
+  outline: none;
+  height: 40px;
+  margin-top: 0.3em;
+  color: #ECC8AF;
+  background: rgba(89,89,75, 0.5);
+  font-size: 20px;
+  transition: all 0.15s ease-in-out;
+  transform: scale(1);
+  line-height: 40px;
+  border-radius: 20px;
+  display: inline-block;
+  padding: 0 20px;
+}
+
+.form-input[type="submit"]:hover {
+  transform: scale(1.03);
+  cursor: pointer;
+  background: rgba(89,89,75, 0.8);
+}
+
+/*** ACCORDION ***/
+
+.accordion-container {
+  background: transparent;
+  color: #ECC8AF;
+}
+
+.accordion {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+  border-bottom: 0.05px solid #ECC8AF;
+  transition: all 0.15s ease-in-out;
+  transform: scale(1);
+}
+
+.accordion-header:hover {
+  font-size: 120%;
+  cursor: help;
+}
+
+.accordion-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.accordion-header:after {
+  content: '\02795';
+}
+
+.accordion-header.active {
+  font-size: 120%;
+}
+
+.accordion-header.active:after {
+  content: "\02796";
+}
+
+.accordion-text-container {
+  height: 0;
+  overflow: hidden;
+  transition: heigt 0.3s ease;
+  font-size: 13px;
+  text-align: justify;
+}
+
+.accordion-text-container.active {
+  height: auto;
+  font-size: 13px;
+}
+
+.accordion-text {
+  margin: 0;
+  padding: 0 5px 15px 5px;
+}
+
+/*** FOOTER ***/
+
+.footer {
+  display: flex;
+  justify-content: center;
+  color: #ECC8AF;
+  margin: 30px;
+}
+
+/*** MEDIA QUERIES ***/
+
+@media only screen and (min-width: 768px) {
+
+  .hero-headline {
+    font-size: 65px;
+    font-weight: 700;
+  }
+
+  .form-container,
+  .accordion-container {
+    width: 90%;
+    margin-top: 50px;
+    max-width: 70vw;
+  }
+
+  .accordion-container {
+    background: #ECC8AF;
+    color: #3E4E50;
+  }
+
+  .form-input[type="submit"] {
+    width: 200px;
+  } 
+}
+
+@media only screen and (min-width: 1024px) {
+
+  .hero-container {
+    max-height: 100vh;
+  }
+
+  .form-container,
+  .accordion-container {
+    max-width: 60vw;
+  }
+}
```

Comment:
Good to see that you don't have to write that much code to still have a completely responsive website 🤩